dependencies/maven/artifacts.snapshot

```diff
@@ -137,12 +143,14 @@
 @maven//:org_objenesis_objenesis
 @maven//:org_objenesis_objenesis_2_5
 @maven//:org_slf4j_jcl_over_slf4j
-@maven//:org_slf4j_jcl_over_slf4j_1_7_20
+@maven//:org_slf4j_jcl_over_slf4j_1_7_32
```

Comment:
This is to address vulnerability CVE-2021-44228 that was recently exposed in Log4J.